LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License
+
+Copyright (c) 2010-2018 Google, Inc. http://angularjs.org
```

Comment:
Copyright google?

Reply:
Ack. missed that. So did the devs of project I got that from. Pulling from official license page.